README.md

```diff
@@ -1148,20 +1148,43 @@ condition](#safe-assignment-in-condition).
 
   ```Ruby
   class Person
+    # bad
+    attr_reader(:name, :age)
+    # good
     attr_reader :name, :age
 
-    # omitted
+    # body omitted
   end
 
+  # bad
+  temperance = Person.new 'Temperance', 30
+  # good
   temperance = Person.new('Temperance', 30)
-  temperance.name
 
+  # bad
+  puts(temperance.age)
+  # good
   puts temperance.age
 
+  # bad
+  x = Math.sin y
+  # good
   x = Math.sin(y)
+
+  # bad
+  array.delete e
+  # good
   array.delete(e)
 
-  bowling.score.should == 0
+  # bad
+  expect(bowling.score).to eq 0
+  # good
+  expect(bowling.score).to eq(0)
+
+  # bad
+  x+(y)
```

Comment:
Not sure what this illustrates, but I don't think it belongs here. Seems like it illustrates redundant parens.